extensions/opentelemetry/opentelemetry-async-mutiny/runtime/src/main/java/io/quarkus/opentelemetry/async/mutiny/runtime/tracing/OpenTelemetryUniInterceptor.java

```diff
@@ -0,0 +1,26 @@
+package io.quarkus.opentelemetry.async.mutiny.runtime.tracing;
+
+import io.opentelemetry.context.Context;
+import io.smallrye.mutiny.Uni;
+import io.smallrye.mutiny.infrastructure.UniInterceptor;
+import io.smallrye.mutiny.subscription.UniSubscriber;
+
+public class OpenTelemetryUniInterceptor implements UniInterceptor {
```

Comment:
Same as for the Multi version - it has a massive performance cost.

Reply:
See comment above